packages/js/src/analysis/TermDataCollector.js

```diff
@@ -65,31 +63,15 @@ TermDataCollector.prototype.getData = function() {
 	};
 };
 
-/**
- * Returns the title from the DOM.
- *
- * @returns {string} The title.
- */
-TermDataCollector.prototype.getTitle = function() {
-	return document.getElementById( "hidden_wpseo_title" ).value;
-};
-
 /**
  * Returns the keyword from the DOM.
  *
  * @returns {string} The keyword.
  */
 TermDataCollector.prototype.getKeyword = function() {
-	var elem, val;
-
-	elem = document.getElementById( "hidden_wpseo_focuskw" );
-	val = elem.value;
-	if ( val === "" ) {
-		val = document.getElementById( "name" ).value;
-		elem.placeholder = val;
-	}
+	const elem = document.getElementById( "hidden_wpseo_focuskw" );
 
-	return val;
+	return elem.value;
```

Comment:
Remove the step to set the term name as the keyphrase when the `hidden_wpseo_focuskw` is empty